pytensor/tensor/slinalg.py

```diff
@@ -1,3 +1,4 @@
+import functools as ft
```

Comment:
Nitpick, is this a common alias?

```suggestion
from functools import reduce
```

Reply:
Not sure, I think I saw it in the pytensor code once. But you're right that for base library it's cleaner to just import the functions directly.